modules/apps/translation/translation-google-cloud-translator/bnd.bnd

```diff
@@ -1,6 +1,6 @@
-Bundle-Name: Liferay Translation Google Cloud Translator
-Bundle-SymbolicName: com.liferay.translation.google.cloud.translator
-Bundle-Version: 1.0.11
+Bundle-Name: Liferay Translation Translator Google Cloud
+Bundle-SymbolicName: com.liferay.translation.translator.google.cloud
```

Comment:
looking at the failling poshi tests there is calls to the old module `com.liferay.translation.google.cloud.translator` (as on configuration on DMNavigator.macro) and other calls that maybe must be changed.

Reply:
dupe!

Reply:
Done.